protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoLiteUtils.java

```diff
@@ -48,12 +54,27 @@
 
   private static final int BUF_SIZE = 8192;
 
+  private static final String JAVA_VERSION = getSystemProperty("java.specification.version");
+
+  /**
+   * Assume Java 9+ if it isn't Java 7 or Java 8.
+   */
+  @VisibleForTesting
+  static final boolean IS_JAVA9_OR_HIGHER =
+      !"1.7".equals(JAVA_VERSION) && !"1.8".equals(JAVA_VERSION);
```

Comment:
How about something like
```java
static {
  boolean isJava9 = true;
  try {
    Class.forName("java.lang.StackWalker");
  } catch (ClassNotFoundException cnfe) {
    isJava9 = false;
  }
  IS_JAVA9_OR_HIGHER = isJava9;
}
```

Reply:
Okay, seems to be simpler and guess it should work equally well. Changed.